src/app/shared/contextmenu/context-menu/context-menu.component.ts

```diff
@@ -105,6 +111,28 @@ export class ContextMenuComponent implements AfterViewInit, OnDestroy {
       domElem.remove();
       contextElement.style.visibility = 'visible';
       document.body.appendChild(domElem);
+
+      // create a dummy node to prevent the browser from capturing focus when tabbing out of the context menu
+      const dummyNode = document.createElement('input');
+      // hide the input immediately on render
+      dummyNode.setAttribute('id', 'contextMenuDummyInput');
+      document.body.appendChild(dummyNode);
+
+      // focus the first element in the context menu for accessibility
+      const firstElement = domElem.querySelector('li');
+      firstElement.focus();
+
+      // listen for the user to tab out of the context menu and close the menu
+      domElem.querySelectorAll('li:first-child, li:last-child').forEach(el => {
+
+        el.addEventListener('blur', (event: FocusEvent) => {
+
+          if (!event.relatedTarget || (event.relatedTarget as HTMLElement).parentElement !== firstElement.parentElement) {
+            this.close.emit();
+          }
+        });
+      });
+
```

Comment:
@nrwinner can we add a doc comment here to describe this pattern a little more thoroughly?